File: Python PowerPoint Examples/ShapesApiDemo/getSlideShape.py
import spirecloudpowerpoint
from spirecloudpowerpoint.configuration import *

appId = "your id"
appKey = "your key"
baseUrl = "https://api.e-iceblue.cn"
configuration = Configuration(appId, appKey,baseUrl)
api= spirecloudpowerpoint.api.shapes_api.ShapesApi(configuration)

fileName = "SlideShapes.pptx"
password = ""
storage = ""
inputFolder = "input"
slide_index = 0
shape_index = "3"
result = api.get_slide_shape(fileName, slide_index, shape_index,password=password, storage=storage,folder=inputFolder)

File: JavaScript PowerPoint Examples/ShapesApiDemo/getSlideShape.js
(function (){
    var appId = "your ID";
    var appKey = "your Key";
    var baseUrl = "https://api.e-iceblue.cn";

    var SpirecloudPowerpoint = require('../../src/index');
    var apiClient = new SpirecloudPowerpoint.ApiClient(appId, appKey,baseUrl);
    var instance = new SpirecloudPowerpoint.ShapesApi(apiClient);
    
    var name = 'SlideShapes.pptx';
    var folder = 'input';
    var slideIndex = 0;
    var shapeIndex = 0;
    var opt = { 'folder': folder };
    var response = instance.getSlideShape(name, slideIndex, shapeIndex, opt,
     function (error, data, response) {
        if (error) throw error;
     });
})();